tests/unit_tests/c_wrappers/integration/test_parameter_sample_types.py

```diff
@@ -108,6 +108,58 @@ def test_gen_kw(tmpdir, config_str, expected, extra_files, expectation):
             )
 
 
+@pytest.mark.integration_test
+@pytest.mark.parametrize(
+    "config_str, expected, extra_files, expectation, relpath",
+    [
+        (
+            "GEN_KW KW_NAME template.txt somepath/kw.txt prior.txt\nRANDOM_SEED 1234",
+            "MY_KEYWORD -0.881423",
+            [],
+            does_not_raise(),
+            "somepath/",
+        ),
+        # This test was added to show current behaviour for Ert.
+        # If absolute paths should be possible to be used like this is up for debate.
+        (
+            "GEN_KW KW_NAME template.txt /tmp/somepath/kw.txt prior.txt INIT_FILES:custom_param%d",
```

Comment:
Can probably simplify this, or is there a reason you would like to use `INIT_FILES`? Does not seem to be present in the test this is replacing? Is it correct that you want to test `somepath/` and `tmp/somepath`?

Reply:
Yes. The original issue was that ert did not store the files in the appropriate location. We need to verify that the file is at that specific place.